cocos/ui/UIEditBox/UIEditBoxImpl-common.cpp

```diff
@@ -414,8 +413,7 @@ void EditBoxImplCommon::editBoxEditingChanged(const std::string& text)
 }
 
 Size ui::EditBoxImplCommon::applyPadding(const Size& sizeToCorrect) const {
```

Comment:
after the discuss, I think make it to __static local function__ is better, it has less space occupy when existing more than one EditBox, and have no outer effect.  you could refer to `static Rect getRect(Node * pNode)` in `UIEditBox.cpp`, both are similar.